files/build_templates/per_namespace/database.service.j2

```diff
@@ -3,9 +3,11 @@ Description=Database container
 {% if multi_instance == 'true' %}
 Requires=database.service
 After=database.service
+Upholds=process-reboot-cause.service
```

Comment:
@xincunli-sonic isn't this redundant? We already added at line 10

Reply:
You are correct, removed.